coil-base/src/main/java/coil/EventListener.kt

```diff
@@ -30,50 +29,41 @@ import coil.transition.TransitionTarget
 interface EventListener : ImageRequest.Listener {
 
     /**
-     * Called immediately after the request is dispatched.
+     * @see ImageRequest.Listener.onStart
      */
     @MainThread
-    fun onDispatch(request: ImageRequest) {}
+    override fun onStart(request: ImageRequest) {}
 
     /**
-     * Called before any [Mapper]s and/or [MeasuredMapper]s are called to convert the request's data.
-     *
-     * @param input The data that will be converted.
+     * Called before [SizeResolver.size].
      */
     @MainThread
-    fun mapStart(request: ImageRequest, input: Any) {}
+    fun resolveSizeStart(request: ImageRequest) {}
```

Comment:
`input` is now always equal to `request.data` now that `request.data` is not null.